app/models/user.rb

```diff
@@ -38,13 +39,13 @@ class User < ActiveRecord::Base
 
   validates_with UniqueUsernameValidator, on: :create
   validates_format_of :username, with: /^[A-Za-z\d_\-]+$/
+  validates_format_of :email, with: /@/
 
   before_create :create_drupal_user
   before_save :set_token
   after_destroy :destroy_drupal_user
 
   def create_drupal_user
-    self.bio = "" # needed to set a default bio value of ""
```

Comment:
I guess we should add **self.bio=""** here or in **user_test.rb** when we create a new user ☺️ .

Reply:
I tried changing it to self.bio ||= "" instead of moving it to the controller. Let's see how that does!!!

Reply:
Because it had been overwriting the given bio in tests.